testsuite/smoke-test/src/operational_tooling.rs

```diff
@@ -606,47 +599,65 @@ async fn test_extract_peers_from_keys() {
 
 #[tokio::test]
 async fn test_generate_key() {
-    let op_tool = OperationalTool::test();
     let path = TempPath::new();
     path.create_as_file().unwrap();
+    let pub_path = append_file_extension(path.as_ref(), ".pub").unwrap();
 
     // Base64
-    let expected_key = op_tool
-        .generate_key(KeyType::X25519, path.as_ref(), EncodingType::Base64)
-        .await
-        .unwrap();
-    assert_eq!(
-        expected_key,
-        x25519::PrivateKey::try_from(
-            base64::decode(fs::read(path.as_ref()).unwrap())
-                .unwrap()
-                .as_slice()
-        )
-        .unwrap(),
-    );
+    let (priv_key, pub_key) =
```

Comment:
The pedantic person inside of me would make a separate unit test for each of the key encodings we're testing in here 😄

Reply:
Yeah, I wasn't going to rewrite it for multiple tests though :P